tests/filecheck/dialects/qssa/ops.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+%q0, %q1 = qssa.alloc<2>
+
+// CHECK: %q0, %q1 = qssa.alloc<2>
+
+%0 = qssa.h %q0
+
+// CHECK-NEXT: %0 = qssa.h %q0
+
+%q2:2 = qssa.cz %q1 %0
```

Comment:
I think it's customary for custom syntax to have commas between operands, so I'd suggest adding them if you don't have a reason to omit them.